pkg/operator/ceph/file/filesystem.go

```diff
@@ -156,6 +176,9 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 					fmt.Sprintf(". %v", err),
 			)
 		}
+		if err := SetPoolSize(f, context, spec); err != nil {
+			return errors.Wrap(err, "failed to set pools size")
```

Comment:
Should this report the filesystem name for users?

Reply:
it just reports that whether or not the pool size were able to set or not?.